### PR TITLE
Updated docker-compose files

### PR DESCRIPTION
**Pull request by isaacrivriv**
Changed the sample docker-compose files to include both user:pass and apikey in comments so that clients can uncomment and fill whichever they have.

### CODE REVIEW

File: docker/docker-compose-self-service.yml [hunk 1]
```diff
@@ -33,13 +37,21 @@ services:
       - MEDIA_RELAY_WS_PORT=8080
 
       # Creds for Bluemix STT
-      - WATSON_STT_USERNAME=
-      - WATSON_STT_PASSWORD=
+      #If your account is associated with a username and password uncomment WATSON_STT_USERNAME and WATSON_STT_PASSWORD and fill out accordingly
+      #- WATSON_STT_USERNAME=
+      #- WATSON_STT_PASSWORD=
+      #If your account is associated with an apikey unomment WATSON_STT_APIKEY and WATSON_STT_TOKEN_SERVICE_PROVIDER_URL and fill apikey accordingly
```
Comment (jfmartinez) [inline docker/docker-compose-self-service.yml]: Typo: unomment -> uncomment

File: docker/docker-compose-self-service.yml [hunk 2]
```diff
@@ -33,13 +37,21 @@ services:
...
+      #- WATSON_STT_TOKEN_SERVICE_PROVIDER_URL=https://iam.bluemix.net/identity/token
       - WATSON_STT_URL=
 
       # Creds for Bluemix TTS
-      - WATSON_TTS_USERNAME=
-      - WATSON_TTS_PASSWORD=
+      #If your account is associated with a username and password uncomment WATSON_TTS_USERNAME and WATSON_TTS_PASSWORD and fill out accordingly
+      #- WATSON_TTS_USERNAME=
+      #- WATSON_TTS_PASSWORD=
+      #If your account is associated with an apikey use these and comment WATSON_TTS_APIKEY and WATSON_TTS_TOKEN_SERVICE_PROVIDER_URL and fill apikey accordingly
```
Comment (jfmartinez) [inline docker/docker-compose-self-service.yml]: Typo: comment -> uncomment

File: docker/docker-compose-self-service.yml [hunk 3]
```diff
@@ -13,8 +13,12 @@ services:
 
       # Uncomment and specify when the Conversation API is used
       - WATSON_CONVERSATION_WORKSPACE_ID=
-      - WATSON_CONVERSATION_USERNAME=
-      - WATSON_CONVERSATION_PASSWORD=
+      #If your account is associated with a username and password uncomment WATSON_CONVERSATION_USERNAME and WATSON_CONVERSATION_PASSWORD and fill out accordingly
+      #- WATSON_CONVERSATION_USERNAME=
+      #- WATSON_CONVERSATION_PASSWORD=
+      #If your account is associated with an apikey use these and comment WATSON_CONVERSATION_APIKEY and WATSON_CONVERSATION_TOKEN_SERVICE_PROVIDER_URL and fill apikey accordingly
```
Comment (jfmartinez) [inline docker/docker-compose-self-service.yml]: Typo: comment -> uncomment

**Comment by isaacrivriv** [comment]
My bad I thought you made the changes. I fixed them and made another commit to the branch. Not sure if thats the correct way to do it.

**Comment by jfmartinez** [comment]
yup, that's good - thanks!

**Pull request merged**